historicalData/storeRecordsInTimestream.ts

```diff
@@ -4,27 +4,27 @@ import {
 	WriteRecordsCommand,
 	type _Record,
 } from '@aws-sdk/client-timestream-write'
-import { logger } from '../lambda/util/logger.js'
-
-const log = logger('storeRecordsInTimestream')
+import type { Logger } from '@aws-lambda-powertools/logger'
```

Comment:
Unrelated change

Reply:
`historicalData/storeRecordsInTimestream.ts` is used in e2e test to store messages into Timestream, https://github.com/hello-nrfcloud/backend/pull/76/files#diff-73d68b7919e0e0232209be295615541da24bff65aceb4bd6eda55499170e54fcR211. Then the logger will print log message and it messes up the feature runner output. Consequently, `console-runner` cannot parse and throw error.

Reply:
Ah, I see.